spec/dsl_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+
+RSpec.describe "Pundit RSpec DSL" do
+  before do
+    RSpec.configure do |config|
+      config.include ::Pundit::RSpec::DSL
+    end
+  end
+
+  let(:block) { proc { "block content" } }
+
+  it "calls describe with the correct metadata and without :focus" do
+    expected_metadata = { permissions: %i[:item1, :item2], caller: instance_of(Array) }
```

Comment:
Test failures probably already caught this, but `%i` shouldn't have a comma `,` nor colon `:` 🙂